src/Mod/Assembly/AssemblyGlobal.h

```diff
@@ -0,0 +1,47 @@
+/***************************************************************************
+ *   Copyright (c) 2021 Werner Mayer <wmayer[at]users.sourceforge.net>     *
```

Comment:
I'd update the copyright on this one, even though it's trivial. Please use the template from the Developer's Handbook: https://freecad.github.io/DevelopersHandbook/codeformatting/